packages/jsii-pacmak/test/python.test.ts

```diff
@@ -113,6 +113,17 @@ test('list with multiline text', () => {
   );
 });
 
+test('escape character escaping', () => {
+  converts(
+    [
+      'For example, if you specify \\N, BigQuery interprets \\N as a null value',
```

Comment:
I'm not certain if I'm doing something wrong with my test or the markdown parser is splitting into multiple nodes and the change doesn't actually work.
The literal string I'm hoping to test is `For example, if you specify "\N", BigQuery interprets "\N" as a null value`

Reply:
Seems like it is a separate node.